app/scripts/controllers/change-password-v2-controller.js

```diff
@@ -54,7 +54,7 @@ angular.module('stellarClient').controller('ChangePasswordV2Ctrl', function($sco
 
     var params = {
       server: Options.WALLET_SERVER+'/v2',
-      username: $stateParams.username.toLowerCase(),
+      username: $stateParams.username.toLowerCase()+'@stellar.org',
```

Comment:
We have this expression (`+'@stellar.org'`) littered all over the client codebase now.  At least 5 instances on master, and now 2 more here.

Let's extract this into a helper method.